components/bl-data-grid-component/src/use-styles.js

```diff
@@ -6,16 +6,21 @@ export const useStyles = (style, width, height) => useMemo(() => {
   return { ...style, ...validStyles, flexShrink: 0 };
 }, [style, width, height]);
 
-const validateStyles = styles => {
-  return Object.entries(styles).reduce((acc, [propertyName, value]) => {
+const validateStyles = dimensions => {
+  const numbersOnlyRegex = /^\d+$/;
```

Comment:
```suggestion
  const ONLY_NUMERIC_REGEX = /^\d+$/;
```
And move out from function